infrastructure/tf_backend/variables.tf

```diff
@@ -16,7 +16,16 @@ variable "billing_code" {
   type = string
 
   validation {
-    condition     = var.billing_code != "<Fill Billing Code>"
+    condition     = var.billing_code != ""
     error_message = "You must provide your billing code in tf_backend.auto.tfvars."
   }
 }
+
+variable "aws_region" {
```

Comment:
This seems to be how you declare terraform variables so you can access it in other scripts :)